### PR TITLE
Support zkSync

### PR DESCRIPTION
**Pull request by hungdoansy**
# Changes
- Add zkSync to About page
- Add zkSync to chain select
- Fix bug relating to Add liquidity using only 1 token: https://team-kyber.atlassian.net/browse/KGP-1279

# TODO
- [x] Remove console.log
- [x] Resolve TODO
- [x] Revert aggregator API
- [x] Revert .env

### CODE REVIEW

File: src/components/SearchModal/bridge/CurrencyListBridge.tsx [hunk 1]
```diff
@@ -47,7 +47,10 @@ const CurrencyListBridge = memo(function CurrencyListV2({
       const isSelected = isOutput
         ? tokenInfoOut?.sortId === currency?.multichainInfo?.sortId
         : selectedCurrency?.equals(currency)
-      const handleSelect = () => onCurrencySelect(currency)
+      const handleSelect = () => {
+        console.log({ currency })
```
Comment (nguyenhoaidanh) [inline src/components/SearchModal/bridge/CurrencyListBridge.tsx:51]: rm

File: src/constants/tokens.ts [hunk 2]
```diff
@@ -259,6 +261,11 @@ export const SUPER_STABLE_COINS_ADDRESS: { [chainId in ChainId]: string[] } = {
     '0x94b008aA00579c1307B0EF2c499aD98a8ce58e58', // usdt
     '0x7F5c764cBc14f9669B88837ca1490cCa17c31607', // usdc
   ],
+  // TODO: fill here
+  [ChainId.ZKSYNC]: [
```
Comment (nguyenhoaidanh) [inline src/constants/tokens.ts:265]: todo

File: src/hooks/useKyberSwapConfig.ts [hunk 3]
```diff
@@ -65,7 +64,7 @@ const parseGlobalResponse = (
   chainId: ChainId,
 ): KyberswapGlobalConfig => {
   const data = responseData?.data?.config
-  const aggregatorDomain = data?.aggregator ?? DEFAULT_AGGREGATOR_API
+  const aggregatorDomain = 'https://aggregator-api.kyberswap.com'
```
Comment (nguyenhoaidanh) [inline src/hooks/useKyberSwapConfig.ts:67]: revert

File: src/utils/index.ts [hunk 4]
```diff
@@ -523,6 +523,11 @@ export const getLimitOrderContract = (chainId: ChainId) => {
   return ENV_KEY === 'production' || ENV_KEY === 'staging' ? production : development
 }
 
+export const getCrossChainSupport = (chainId: ChainId) => {
```
Comment (nguyenhoaidanh) [inline src/utils/index.ts:526]: unused

File: .env.adpr [hunk 5]
```diff
@@ -1,45 +1,42 @@
...
-VITE_PRICE_ALERT_API=https://price-alert.dev.kyberengineering.io/api
-VITE_BFF_API=https://kyberswap-bff.dev.kyberengineering.io/api
+VITE_ENV=staging
+VITE_OAUTH_CLIENT_ID=bce9bf52-b788-4c25-9c6c-a07dff080403
+VITE_BFF_API=https://kyberswap-bff.stg.kyberengineering.io/api
 VITE_KYBER_AI_REFERRAL_ID=1
-VITE_KYBER_AI_TOPIC_ID=14
+VITE_KYBER_AI_TOPIC_ID=11
 
-VITE_BUCKET_NAME=ks-setting-a3aa20b7
```
Comment (nguyenhoaidanh) [inline .env.adpr:45]: remember rever before merge

File: src/constants/index.ts [hunk 6]
```diff
@@ -291,6 +291,37 @@ export const TRANSACTION_STATE_DEFAULT: TransactionFlowState = {
...
+        ChainId.AVAXMAINNET,
+        ChainId.FANTOM,
+        ChainId.CRONOS,
+        ChainId.ARBITRUM,
+        ChainId.BTTC,
+        ChainId.VELAS,
+        ChainId.AURORA,
+        ChainId.OASIS,
+        ChainId.OPTIMISM,
+        // ChainId.ZKSYNC,
```
Comment (nguyenhoaidanh) [inline src/constants/index.ts:322]: yes or no

File: src/pages/BuyCrypto/index.tsx [hunk 7]
```diff
@@ -207,6 +207,8 @@ function BuyCrypto() {
     [ChainId.VELAS]: 'velasevm',
     [ChainId.SOLANA]: 'solana',
 
+    // TODO: check later
+    [ChainId.ZKSYNC]: null,
```
[the remark this answers is not in the document]
Comment (viet-nv) [reply src/pages/BuyCrypto/index.tsx:211]: null is ok

File: src/constants/tokens.ts [hunk 8]
```diff
@@ -512,6 +521,8 @@ export const USDT: { [chainId in ChainId]: Token } = {
     'USDT',
     'Tether USD',
   ),
+  // TODO(viet-nv): this address doesn't exist. check again
```
Comment (nguyenhoaidanh) [inline src/constants/tokens.ts:524]: todo

Comment (viet-nv) [reply src/constants/tokens.ts:524]: this todo is fine. Let me refactor it later

**Pull request merged**